packages/common/src/types.ts

```diff
@@ -30,6 +30,7 @@ export type MinimalDonorDetailsForAppointment = {
   lastName: string;
   phoneNumber: string;
   bloodType: BloodType;
+  countryIdNumber?: string;
```

Comment:
I think `idNumber` is better - what do you think?

Reply:
I think idNumber sounds like the id of the document..